common/setopt.c

```diff
@@ -52,6 +52,9 @@ AddSetOpt(
         BAIL_ON_TDNF_ERROR(dwError);
     }
 
+    dwError = AddSetOptWithValues(pCmdArgs, CMDOPT_KEYVALUE, pCmdOpt->pszOptName, pCmdOpt->pszOptValue);
```

Comment:
Please break this line.

Reply:
why ?

Reply:
Just to make it more readable.

Reply:
not required

Reply:
It's a bug here . It's getting parsed but not stored.

Reply:
> not required

The line is too long (> 80 chars). Please try to keep lines shorter than that.